src/App.styled.ts

```diff
@@ -82,6 +135,12 @@ export const EventLoop = styled(BaseLayoutElement)`
 	overflow: auto;
 	gap: 10px;
 	position: relative;
+
+	@media (max-width: 768px) {
+		grid-column: 1;
+		grid-row: 9;
+    height: 40vh;
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->

💭 thought:
I like the idea of putting all sections in 1 column.
However, it leads to a long UI with at least 150vh + margins/paddings.
This means that users won't be able to see all the actions on one screen (viewport), so they will definitely miss something. 

If you have any idea on how to make it more UX-friendly we can discuss it and address it in the separate PR.

Reply:
@vault-developer  I have tried something here. Please have a look.
On clicking of the "Run" button, the page scrolls to bring "Event Loop" into focus.

This doesn't solve the problem of user not being able to see all actions on one screen.
That issue remains remains. It can be resolved by reducing the height of  each containers like webApi, taskQueue.

But that would affect readability when flow reaches those container. It's a tricky problem to solve for now.

So, as a logical first step, I am bringing the user's attention to Event Loop when he clicks on "Run" button.

What do you think ?

Reply:
Thank you for the update.
I think this solution is good enough for now, I will create a separate issue to investigate more on how we can put everything on one screen 👍